### PR TITLE
De-TS/JSPM deploy radiator

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

converts deploy radiator to use:
- es6 (the src is just the result of `tsc -m es2015 main.ts`)
- global `package.json`
- npm
- webpack
## What is the value of this and can you measure success?
- simplifies module management 
- begins process of globalising code standard
- introduces webpack
- faster to build and install
## Does this affect other platforms - Amp, Apps, etc?

no
## Screenshots

screen shot 2016-07-13 at 12 14 42

### CODE REVIEW

File: static/src/deploys-radiator/app/render.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                ])))
+                .toList())
+        ]
+    ]);
+};
+const renderPage = ([codeDeploys, prodDeploys], [latestCodeDeploy, oldestProdDeploy], commits) => {
+    const isInSync = oldestProdDeploy.build === latestCodeDeploy.build;
+    return h('.row#root', {}, [
+        h('h1', [
+            `Status: ${isInSync ? 'in sync. Ship it!' : 'out of sync.'}`
```
Comment (desbo) [inline static/src/deploys-radiator/app/render.js]: I always wondered why it says `Ship it` when it's in sync, doesn't that mean there's nothing to ship?

**Comment by TBonnin** [comment]
No more Typescript @OliverJAsh is going to be sad 😁

File: static/src/deploys-radiator/app/render.js (hunk 1 above)
Comment (sndrs) [inline static/src/deploys-radiator/app/render.js]: that's a good point! i guess saying you can merge, the queue is empty?

**Comment by sndrs** [comment]
well how knows...maybe. but it's not moved across to the main app so far, so un-siloing in this instance

**Comment by sndrs** [comment]
was going to pretty up the CSS a little, but @rich-nguyen already said he'd revert any attempt to change that :)

**Comment by OliverJAsh** [comment]
If the software is still being used then awesome :-)

**Comment by sndrs** [comment]
just timed `make reinstall`:

before `1:51`
after `1:09`

**Comment by desbo** [comment]
👍

**Pull request merged**